src/styles/public/navigation-ribbon.scss

```
@@ -0,0 +1,110 @@
+/* stylelint-disable no-descending-specificity */
+/* stylelint-disable selector-class-pattern */
+
+/*
+ *  Ribbon Menu Template Part.
+ */
+.is-style-ribbon-menu {
+
+	.is-offscreen {
+		display: inherit;
+	}
+
+	.has-modal-open {
+		position: fixed !important;
+	}
+
+	nav {
+		padding: 0.75rem;
+		justify-content: flex-end;
+
+		.current-menu-item {
+			background-color: var(--wp--preset--color--primary-brand);
+		}
+
+		@media (min-width: $breakpoint-sm) {
+			padding: 0;
+			justify-content: flex-start;
+		}
+	}
+
+	select.pll-switcher-select {
+		border: 0;
+		color: currentcolor;
+	}
+}
+
+body.has-ribbon-menu {
+
+	.site-header {
+
+		nav {
+			padding: 1rem 2rem;
+
+			@media (min-width: $breakpoint-sm) {
+				padding: inherit;
+			}
+		}
+	}
+
+	.wp-block-navigation {
+
+		a {
+
+			&:hover,
+			&:focus {
+				background-color: rgba($black, 33%);
+			}
+		}
+
+		&.has-background {
+
+			.has-child {
+
+				.wp-block-navigation {
+
+					&__submenu-container {
```

is this supposed to be (if so make sure you update all the instances)
```suggestion
					&.__submenu-container {
```

or is this actually` .wp-block-navigation__submenu-container` if so i guess i just learned something new :)

---

It's your new thing learned today. You officially can go back to bed. ;)

---

too bad i didn't find it this morning, i'm done work for the day, how does that help me now.